tests/components/sma/test_sensor.py

```diff
@@ -10,7 +10,7 @@
 
 async def test_sensors(hass, init_integration):
     """Test states of the sensors."""
-    state = hass.states.get("sensor.current_consumption")
+    state = hass.states.get("sensor.grid_power")
```

Comment:
```suggestion
    state = hass.states.get("sensor.current_consumption")
```
This looks like a change, that are outside the version bump.

Reply:
This is related to the version bump. The current_consumption and total_consumption sensors have been removed since they were not working.

Reply:
ok